book/src/snapshot-restore-feature.md

```diff
@@ -99,7 +99,7 @@ kubectl create -f restore.yaml
 ## Snapshot Alpha
 ### Snapshot APIs
 
-Similar to the API for managing [Kubernetes Persistent Volumes](https://v1-16.docs.kubernetes.io/docs/concepts/storage/persistent-volumes/), the Kubernetes Volume Snapshots introduce three new API objects for managing snapshots: `VolumeSnapshot`, `VolumeSnapshotContent`, and `VolumeSnapshotClass`. See [Kubernetes Snapshot documentation](https://v1-16.docs.kubernetes.io/docs/concepts/storage/volume-snapshots/) for more details.
+Similar to the API for managing [Kubernetes Persistent Volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/), the Kubernetes Volume Snapshots introduce three new API objects for managing snapshots: `VolumeSnapshot`, `VolumeSnapshotContent`, and `VolumeSnapshotClass`. See [Kubernetes Snapshot documentation](https://kubernetes.io/docs/concepts/storage/volume-snapshots/) for more details.
```

Comment:
This mention is under snapshot `alpha` section, so we have to revisit this and also there are references below to external snapshotter 1.2 and alpha APIs..

Reply:
@humblec the link mentioned in `snapshot alpha` section which refer to v1.16 of `k8s.io persistent volume` docs which i am not able to get in offical site i.e https://kubernetes.io/.Should i mentioned here a specific version of it?

Reply:
Since alpha API is not widely used anymore, I would leave the link as-is ; or if you really want to put to that section you can refer to : https://github.com/kubernetes/website/blob/snapshot-initial-v1.16/content/en/docs/concepts/storage/volume-snapshots.md